### PR TITLE
Add labels section to the service form modal

### PR DESCRIPTION
**Pull request by Poltergeist**
This adds the functionality to add Labels in the create modal.

image
image

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @philipnrmn

**Comment by aldipower** [comment]
Daumen hoch! :+1:

File: src/js/schemas/service-schema/Labels.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+var Labels = {
```
Comment (kennyt) [inline src/js/schemas/service-schema/Labels.js]: `let`

File: src/js/schemas/service-schema/Labels.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  title: 'Labels',
+  properties: {
+    labels: {
+      type: 'array',
+      duplicable: true,
+      addLabel: 'Add Label',
+      getter: function (service) {
+        let labels = service.getLabels() || {};
+
+        return Object.keys(labels).reduce(function (memo, key) {
```
Comment (kennyt) [inline src/js/schemas/service-schema/Labels.js]: i feel like this could be better implemented using `map`. what do you think?

**Comment by kennyt** [comment]
after my two super nits, :ship:

File: src/js/schemas/service-schema/Labels.js (hunk 2 above)
Comment (Poltergeist) [inline src/js/schemas/service-schema/Labels.js]: You are right I was missing that.

File: src/js/schemas/service-schema/Labels.js (hunk 1 above)
Comment (Poltergeist) [inline src/js/schemas/service-schema/Labels.js]: changed it to const

File: src/js/schemas/service-schema/Labels.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      addLabel: 'Add Label',
+      getter: function (service) {
+        let labels = service.getLabels() || {};
+
+        return Object.keys(labels).map(function (key) {
+          return {
+            key,
+            value: labels[key]
+          };
+        }, []);
```
Comment (kennyt) [inline src/js/schemas/service-schema/Labels.js]: you don't need the 2nd argument `[]` :D

**Pull request merged**